codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/shapevisitor/AwsSdkToDafnyShapeVisitor.java

```diff
@@ -37,6 +42,7 @@ public class AwsSdkToDafnyShapeVisitor extends ShapeVisitor.Default<String> {
 
   private final boolean isOptional;
   protected boolean isPointerType;
+  public static final Map<MemberShape, String> visitorFuncMap = new HashMap<>();
```

Comment:
Static constants are named in uppercase snake_case, `VISITOR_FUNC_MAP`.

Reply:
This was updated to be a private variable. Not necessarily for private var